components/Loan/List/index.tsx

```diff
@@ -33,25 +31,6 @@ class LoanList extends React.Component<Props> {
               />
             </Box>
         },
-        {
-          header: 'LoanOwner', property: 'loanOwner', align: 'end',
-          render: (l: Loan) => <div>
-            <Box style={{ maxWidth: '150px' }}>
-              <DisplayField
-                copy={true}
-                as={'span'}
-                value={l.ownerOf}
-              />
-            </Box>
-
-          </div>
-        },
-        {
-          header: '', property: '', align: 'end',
-          render: (l: Loan) => <div>
-            {proxies.includes(l.ownerOf.toString()) && <Badge text={'Me'} />}
-          </div>
-        },
```

Comment:
Is this removal intended? How do users find their own loans, e. g. in the demo UI?

Reply:
yes it is intentional :) we don;t need it anymore since our current deployments just have one borrower

Reply:
in the demo ui this would be an issue, you are right. We can reintroduce it, once we have a spec for the new demo ui